### PR TITLE
Bug fixes for set_asset_tag command and adding ancestor_list parameter to parent-child process lineage commands

### PR DESCRIPTION
**Pull request by bkschmoll-uptycs**
## Status
Ready/In Progress/In Hold(Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by anara123** [comment]
@bkschmoll-uptycs Thank you for your contribution. Please sign the CLA first

**Comment by bkschmoll-uptycs** [comment]
Whoops, sorry about that!  I must have missed the previous emails.  I have just signed.

**Comment by anara123** [comment]
@bkschmoll-uptycs please take a look at the build, it fails.
You need to add CHANGELOG.md. Follow: https://github.com/demisto/content/blob/master/docs/release_notes/README.md

File: Integrations/Uptycs/Uptycs.yml [hunk 1]
```diff
@@ -1778,4 +1790,4 @@ script:
   isfetch: true
   runonce: true
   script: '-'
-  type: python
+  type: python3
```
Comment (anara123) [inline Integrations/Uptycs/Uptycs.yml]:
```suggestion
  type: python
  subtype: python3
```

**Comment by anara123** [comment]
@kirbles19 please review the PR.

**Comment by kirbles19** [comment]
@anara123 Done.

**Pull request merged**